stdeb.cfg

```diff
@@ -1,7 +1,4 @@
 [DEFAULT]
-Depends: apache2, apache2-bin (>=2.4.18) | libapache2-mod-proxy-html, ssl-cert, libxml2-dev
-Recommends: python-nmoscommon
-Depends3: apache2, apache2-bin (>=2.4.18) | libapache2-mod-proxy-html, ssl-cert, libxml2-dev
-Recommends3: python3-nmoscommon
+Depends: apache2, apache2-bin (>=2.4.18) | libapache2-mod-proxy-html, ssl-cert, libxml2-dev, ips-reverseproxy-common, python-nmoscommon 
+Depends3: apache2, apache2-bin (>=2.4.18) | libapache2-mod-proxy-html, ssl-cert, libxml2-dev, ips-reverseproxy-common, python3-nmoscommon 
 Build-Depends: apache2-dev, dh-python, dh-systemd
-Package: ips-reverseproxy-common
```

Comment:
Not sure this one is quite right. It seems to depend upon itself now and the 'Package' line has been removed.